.github/cloudbuild/pr-open-to-main-pos.yaml

```diff
@@ -0,0 +1,87 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+timeout: 900s
+steps:
+- id: 'maven-install'
+  name: 'gcr.io/cloud-builders/mvn'
+  dir: 'point-of-sale-app'
+  args: [
+      'clean',
+      'install'
+  ]
+  waitFor: [ '-' ]
+
+# npm install to ensure ui dependencies have no errors
+- id: 'npm-install'
+  name: 'node:16.14.0'
+  dir: 'point-of-sale-app/ui'
+  entrypoint: 'npm'
+  args: [ 'install' ]
+  waitFor: [ '-' ]
+
+# create namespace yaml for the PR
+- id: 'create-namespace.yaml'
+  name: ubuntu
+  entrypoint: bash
+  args:
+  - -c
+  - |
```

Comment:
i see that there is `.github/cloudbuild/namespace.yaml` also in the PR, with the same content as within this step, do we need both?

Reply:
Good catch. It was used whilst testing. I removed it in 7958f4b165db26d51478f9e4302b2b77353b2650